dnSpy/dnSpy/Documents/TreeView/InternetShortcutFileFormat.cs

```diff
@@ -0,0 +1,16 @@
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace dnSpy.Documents.TreeView;
+
+// Helper class to interact with Microsoft' InternetShortcut file format used in .url files
+static class InternetShortcutFileFormat {
```

Comment:
I don't think the comment here is necessary. The class name is already self-explanatory as to what its purpose is ;P

Reply:
Done.
I just wanted to make sure people know its microsoft and its only in url files. Depending on a countries google search index it could be elusive.